ansible_roles/roles/azure_create/tasks/main.yml

```diff
@@ -11,6 +11,12 @@
     copy:
       src: "tf/main_no_net.tf"
       dest: "{{ working_dir }}/tf/main.tf"
+  - name: cat non spot machine terraform
+    shell: "cat {{ config_info.local_run_dir }}/ansible_roles/roles/azure_create/files/tf/vm_spot_set | sed \"s/PRIORITYSPOT//g\" | sed \"s/EVICTIONPOLICY//g\" >> {{ working_dir }}/tf/main.tf" 
+    when: config_info.spot_start_price == 0
+  - name: cat non spot machine terraform
+    shell: "cat {{ config_info.local_run_dir }}/ansible_roles/roles/azure_create/files/tf/vm_spot_set | sed \"s/PRIORITYSPOT/priority              = \\\"Spot\\\"/g\" | sed \"s/EVICTIONPOLICY/eviction_policy       = \\\"Deallocate\\\"/g\" >> {{ working_dir }}/tf/main.tf" 
```

Comment:
Yeah, because have run into numerous instances of it not working, or it works this month and an upgrade happens and they broke it.  So got tired of it, and simply do it this way, works all the time.  If ansible ever stabilizes enough where they do not break things that were working, will move this stuff to use the template roles.